app/views/games/show.html.erb

```diff
@@ -1 +1,49 @@
-<h1><%= @game.name %></h1>
\ No newline at end of file
+
+<%= javascript_include_tag "board" %>
+<div class="col-11 offset 0.5 container">
+  <div class="col-8 offset-2">
+    <h1 class="board-title"><%= @game.name %></h1>
+    <br />
+  </div>
+      
+  <div class="row">
+    <div class="col-2 my sidebar">
+      <%= @game.user.email %>
+    </div>
+    <div class="col-8">
+      <div class="board">
+        
+        <% row = 8 %>
+        <% while row > 0 %>
```

Comment:
you can turn the while into a for loop like for the column... for example with:

8.downto(1).each do |row|
# your code here
end

Reply:
I'll have a look at that but probably won't have time until tomorrow. 
Tried for loop but wasn't working properly when it's inside that <%= stuff, worked fine in a seperate .rb file. Is theer anyway to place the code in a separate .rb file and have the html.erb file run it (like a javascript file) - looks a bit messy sticking a bunch of code in those html files but if that's the rails way I'll carry on.